### PR TITLE
feat: new design for Details panel

### PR DESCRIPTION
**Pull request by sashahonchar**
image
image

### CODE REVIEW

File: packages/components/src/components/DetailsPanelList.vue [hunk 1]
```diff
@@ -96,44 +96,45 @@ export default {
...
 <style scoped lang="scss">
 .v-details-panel-list {
-  list-style-type: none;
-  padding: 0;
-  margin: 0 -1rem 1.5rem;
+  margin-bottom: 1rem;
+  border-radius: $border-radius;
+  padding: 0.5rem;
+  color: $base03;
+  background-color: rgba(0, 0, 0, 0.1);
```
Comment (dustinbyrne) [inline packages/components/src/components/DetailsPanelList.vue:103]: This style is duplicated in many different components. Can it be moved into it's own component? The Figma contains guidelines for what should be componentized.

**Review by dustinbyrne — CHANGES_REQUESTED** [review]
This is looking good. Regarding the style, there's a small border around these containers that's missing in the implementation:
image
image

The container around the list and the parameter value are the same.

File: packages/components/src/components/DetailsPanelList.vue (hunk 1 above)
Comment (sashahonchar) [reply packages/components/src/components/DetailsPanelList.vue]: I moved that unit test to `VsCodeExtension.spec.js` https://github.com/applandinc/appmap-js/pull/550/commits/e5bd8d3fdb624e9b23e5a67afd6d4970b55952f5

File: packages/components/tests/unit/DetailsPanelEvent.spec.js [hunk 2]
```diff
@@ -6,24 +6,6 @@ import { store, SET_APPMAP_DATA } from '@/store/vsCode';
...
-      store,
-    });
-
-    wrapper.findComponent({ ref: 'viewSource' }).trigger('click');
-
-    const rootWrapper = createWrapper(wrapper.vm.$root);
-    const [[location]] = rootWrapper.emitted().viewSource;
-    expect(location).toBe(event.codeObject.location);
-  });
-
```
[the remark this answers is not in the document]
Comment (sashahonchar) [reply packages/components/tests/unit/DetailsPanelEvent.spec.js:26]: I moved that unit test to `VsCodeExtension.spec.js` https://github.com/applandinc/appmap-js/pull/550/commits/e5bd8d3fdb624e9b23e5a67afd6d4970b55952f5

**Pull request closed without merging**